.travis.yml

```diff
@@ -3,7 +3,7 @@
 # TODO: Test against Mono as well, using matrix.include
 language: csharp
 mono: none
-dotnet: 2.1.101
+dotnet: 2.1
```

Comment:
According to https://docs.travis-ci.com/user/languages/csharp/#.NET-Core, Travis actually wants the SDK version here. Maybe try `2.1.300`?

Reply:
Tried that earlier. There are problems, basically. https://github.com/travis-ci/travis-ci/issues/9684 is the relevant Travis issue - I'm sure it'll be resolved swiftly :)